### PR TITLE
S3C-885: Change v2 query auth window to 7 days

### PR DESCRIPTION
**Pull request by nicolas2bert**
https://github.com/scality/S3/pull/1009

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@ironman-machine
SCALITY_FEDERATION_BRANCH=skip/check-upgrade
try

File: tests/unit/auth/v4/queryAuthCheck.js [hunk 1]
```diff
@@ -199,4 +199,32 @@ describe('v4 queryAuthCheck', () => {
...
+        clock.uninstall();
+        assert.deepStrictEqual(res.err, null);
+        assert.strictEqual(res.params.version, 4);
+        done();
+    });
+
+    it('should successfully return v4 and no error if X-Amz-Expires param ' +
+    'is less thant 604800 (7 days)', done => {
+        // Freezes time so date created within function will be Feb 8, 2016
+        // (within 15 minutes of timestamp in request)
```
Comment (LaurenSpiegel) [inline tests/unit/auth/v4/queryAuthCheck.js]: isn't this comment about 15 minutes no longer relevant?

File: tests/unit/auth/v4/queryAuthCheck.js [hunk 2]
```diff
@@ -199,4 +199,32 @@ describe('v4 queryAuthCheck', () => {
         assert.strictEqual(res.params.version, 4);
         done();
     });
+
+    it('should successfully return v4 and no error if X-Amz-Expires param ' +
+    'is 604800 (7 days)', done => {
+        // Freezes time so date created within function will be Feb 8, 2016
+        // (within 15 minutes of timestamp in request)
```
Comment (LaurenSpiegel) [inline tests/unit/auth/v4/queryAuthCheck.js]: same about 15 minutes here?

File: tests/unit/auth/v4/queryAuthCheck.js [hunk 3]
```diff
@@ -199,4 +199,32 @@ describe('v4 queryAuthCheck', () => {
...
+        const clock = lolex.install(1454974984001);
+        const alteredRequest = createAlteredRequest({ 'X-Amz-Expires':
+        604799 }, 'query', request, query);
+        const res = queryAuthCheck(alteredRequest, log, alteredRequest.query);
+        clock.uninstall();
+        assert.deepStrictEqual(res.err, null);
+        assert.strictEqual(res.params.version, 4);
+        done();
+    });
```
Comment (LaurenSpiegel) [inline tests/unit/auth/v4/queryAuthCheck.js]: do we have a test that get an error if more than 7 days?

Comment (nicolas2bert) [reply tests/unit/auth/v4/queryAuthCheck.js]: Yes, it was already there: https://github.com/scality/Arsenal/pull/370/files#diff-df0485e6d360cbf0a80bf3a0a25dd18cR149

File: tests/unit/auth/v4/queryAuthCheck.js (hunk 1 above)
Comment (nicolas2bert) [reply tests/unit/auth/v4/queryAuthCheck.js]: correct, thanks

File: tests/unit/auth/v4/queryAuthCheck.js (hunk 2 above)
Comment (nicolas2bert) [reply tests/unit/auth/v4/queryAuthCheck.js]: correct, thanks

**Comment by nicolas2bert** [comment]
@ironman-machine
clean
try

**Comment by nicolas2bert** [comment]
@ironman-machine
SCALITY_FEDERATION_BRANCH=rel/6.4
try

**Comment by nicolas2bert** [comment]
@ironman-machine
SCALITY_S3_BRANCH=fx/presignedurl
CI_AGGRESSOR_FORCE_BUILD=1
try

**Pull request merged**